### DISCUSSION TITLE
Comentario

### DISCUSSION THREAD

**Discussion by zBONNELESz** [jordyamc/UKIKU#28] [General]
hola, la opción de descargar todo no funciona bien solo descarga donde se selecciono la opción y como recomendación debería de tener una opción para seleccionar y descargar solo ciertos episodios. gracias

**Comment by jordyamc** [comment]
Con descargar todo se refiere a descargar todos los capitulos seleccionados, no a todos los capitulos de anime

**Comment by casasDev** [comment]
Hola buenas. He visto que en alguna clase has usado la interfaz observer para implementar el patrón con el mismo nombre. Esa interfaz está obsoleta desde la versión 7 de java, versión en la que sacaron la clase Propertychangesupport que hace de observable y propertychangeListener que hacer de observer. Por lo demás me parece un proyecto increíble y tienes mis dieces. Mucho ánimo ☺️.

PD.: Me gustaría colaborar en este proyecto, sin ánimo de lucro claro
